umm-spec-lib/test/cmr/umm_spec/test/umm_to_xml_mappings/echo10.clj

```diff
@@ -107,3 +107,64 @@
     {:DOI {:DOI "10.5678/collectiondoi"
            :Authority "doi.org"}
      :AssociatedDOIs nil}))
+
+(deftest echo10-use-constraints-test
+  "Testing the echo10 use constraint translation from umm-c to echo10."
+
+  (testing "echo10 use constraints description test"
+    (let [actual-data {:UseConstraints {:Description "Description"}}
+          result (echo10/umm-c-to-echo10-xml actual-data)
+          echo-result (first (select result "/Collection/UseConstraints"))]
+      (is (= "Description" (value-of echo-result "Description")))))
+
+  (testing "echo10 use constraints LicenseURL test"
+    (let [actual-data {:UseConstraints {:LicenseURL {:Linkage "https://someurl.com"
+                                                     :Protocol "https"
+                                                     :ApplicationProfile "profile"
+                                                     :Name "License URL"
+                                                     :Description "License URL Description"
+                                                     :Function "information"
+                                                     :MimeType "text/html"}}}
+          result (echo10/umm-c-to-echo10-xml actual-data)
+          echo-result (first (select result "/Collection/UseConstraints"))]
+      (is (and (= "https://someurl.com" (value-of echo-result "LicenseURL/URL"))
```

Comment:
nit: rather than doing all fields check in one `is`, do each field in its own `is`. This way you will get the exact field name when the test fails.

Reply:
I have updated the test.